src/testRunner/unittests/publicApi.ts

```diff
@@ -77,3 +77,31 @@ describe("unittests:: Public APIs:: isPropertyName", () => {
         assert.isTrue(ts.isPropertyName(prop), "PrivateIdentifier must be a valid property name.");
     });
 });
+
+describe("unittests:: Public APIs:: getTypeAtLocation", () => {
+    it("works on PropertyAccessExpression in implements clause", () => {

```

Comment:
would this be easier as a fourslash test? I mean, the answer is probably No, but I thought that quickinfo used getTypeAtLocation.

Reply:
Mmmmmaaybe? I didn’t think about quick info. I had hoped that the type baseliner would do the job, but it didn’t drill all the way down to the problematic node. Quick info also seems susceptible to future changes in which nodes it calls `getTypeAtLocation` on, even if it works now. I do wish it was easier to set up this kind of test.